CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change history for ui-eholdings
 
+## [7.2.0] (IN PROGRESS)
```

Comment:
Let's remove all entries for 7.2.0, and move entry about updating NodeJS to 7.1.5

Reply:
Fixed it, thanks!